.github/workflows/integration.yml

```diff
@@ -177,6 +177,111 @@ jobs:
         run: |-
           guardian plan -dir="${DIRECTORY}" -storage="gcs://${GUARDIAN_BUCKET_NAME}"
 
+      # - name: 'Aggregate PR Approver Data'
```

Comment:
Im sure youre thinking of this but, this could be part of the platform impl and generated during policy command.